Dockerfile

```diff
@@ -73,16 +74,18 @@ RUN curl -sL https://github.com/postmodern/chruby/archive/v0.3.9.tar.gz | tar xz
     rm -rf /chruby-* && rm -rf /usr/local/src/* && yum clean all
 
 ## GIT clone manageiq-appliance and self-service UI repo (SSUI)
-RUN git clone --depth 1 https://github.com/ManageIQ/manageiq-appliance.git ${APPLIANCE_ROOT} && \
-git clone --depth 1 https://github.com/ManageIQ/manageiq-ui-self_service.git ${SSUI_ROOT} && \
-ln -vs ${APP_ROOT} /opt/manageiq/manageiq
+RUN mkdir -p ${APP_ROOT} && \
+    mkdir -p ${APPLIANCE_ROOT} && \
+    mkdir -p ${SSUI_ROOT} && \
+    ln -vs ${APP_ROOT} /opt/manageiq/manageiq && \
+    curl -L https://github.com/ManageIQ/manageiq-appliance/tarball/${REF} | tar vxz -C ${APPLIANCE_ROOT} --strip 1 && \
+    curl -L https://github.com/ManageIQ/manageiq-ui-self_service/tarball/${REF} | tar vxz -C ${SSUI_ROOT} --strip 1
 
-## Create approot, ADD miq
-RUN mkdir -p ${APP_ROOT}
-ADD . ${APP_ROOT}
+## Add ManageIQ source from local directory (dockerfile development) or from Github (official build)
+# ADD . ${APP_ROOT}
```

Comment:
@fbladilo The nice thing about using the tarball (especially for the external repos) is that it does not include the `.git` directory, so it will save us some space.  Also nice is that you don't need to remember to do a fresh clone of the source before building to avoid including things like your own local config settings or log files (oops) in the image that you intend to push to docker hub.

I tried to ADD with the path to the repo tarball, but that apparently only expands when the tarball is checked in.  When it isn't checked in, the tarball itself is dropped in the destination directory.  The difference in behavior was very disappointing.

My understanding of the use cases, and please correct me if I'm missing something, is:
- The majority of builds will be automated branch builds on docker hub (no user involved once https://github.com/ManageIQ/manageiq/issues/8132 is resolved)
- A small percentage of builds will be tag builds probably done by @Fryguy, @simaishi, or myself (until we automate that)
- Very rarely developers will try to build the container with locally modified files to troubleshoot a container build issue.